addons/l10n_kz/data/account_account_template_data.xml

```diff
@@ -0,0 +1,2017 @@
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1400" model="account.account.template">
+        <field name="name">Текущие налоговые активы</field>
+        <field name="code">1400</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1500" model="account.account.template">
+        <field name="name">Долгосрочные активы, предназначенные для продажи</field>
+        <field name="code">1500</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_non_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1600" model="account.account.template">
+        <field name="name">Биологические активы</field>
+        <field name="code">1600</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1700" model="account.account.template">
+        <field name="name">Прочие краткосрочные активы</field>
+        <field name="code">1700</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="note"></field>
+    </record>
+    <record id="a1010" model="account.account.template">
+        <field name="name">Денежные средства в кассе</field>
+        <field name="code">1010</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">учитывается движение денежных средств в кассе в национальной и в иностранных валютах</field>
+    </record>
+    <record id="a1020" model="account.account.template">
+        <field name="name">Денежные средства в пути</field>
+        <field name="code">1020</field>
+        <field name="reconcile" eval="False"/>
+        <field name="chart_template_id" ref="l10n_kz.chart_template"/>
+        <field name="user_type_id" ref="account.data_account_type_liquidity"/>
+        <field name="note">учитывается движение денежных средств в пути и конвертации валюты</field>
+    </record>
+    <record id="a1030" model="account.account.template">
+        <field name="name">Денежные средства на текущих банковских счетах</field>
```

Comment:
it's already in the chart template as bank account prefix, so no need of this record

Reply:
The same. Kazakh accountants are used to have `1030` account instead of `1030.something`